.prow/applications-catalog.yaml

```diff
@@ -0,0 +1,480 @@
+presubmits:
+  - name: pre-kubermatic-appcatalog-argocd
+    run_if_changed: "(pkg/ee/default-application-catalog/argocd-app.yaml)"
+    decorate: true
+    clone_uri: "ssh://git@github.com/kubermatic/kubermatic.git"
+    labels:
+      preset-aws-e2e-kkp: "true"
+      preset-kubeconfig-ci: "true"
+      preset-docker-mirror: "true"
+      preset-docker-pull: "true"
+      preset-docker-push: "true"
+      preset-kind-volume-mounts: "true"
+      preset-goproxy: "true"
+      preset-e2e-ssh: "true"
+    spec:
+      containers:
+        - image: quay.io/kubermatic/build:go-1.23-node-20-kind-0.24-4
+          command:
+            - "./hack/ci/run-default-application-e2e-test.sh"
+          args:
+            - "--application-name"
+            - "argocd"
+            - "--application-version"
+            - "v2.10.0"
+          env:
+            - name: KUBERMATIC_EDITION
+              value: ee
+            - name: SERVICE_ACCOUNT_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: e2e-ci
+                  key: serviceAccountSigningKey
+          securityContext:
+            privileged: true
+          resources:
+            requests:
+              memory: 14Gi
+              cpu: 4
+            limits:
+              memory: 18Gi
```

Comment:
I don't think we need that many resources for each of those jobs. Can you please try if 4GB memory / 2 CPUs as requests and 6GB as memory limit suffice?